adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBNativeRenderer.m

```diff
@@ -88,26 +88,41 @@ - (void)renderNativeAdForAdConfiguration:
                  strongSelf->_nativeAd = nativeAd;
                  strongSelf->_nativeAd.delegate = strongSelf;
                  strongSelf->_nativeAd.rootViewController = adConfiguration.topViewController;
-
-                 if (strongSelf->_loadCompletionHandler) {
-                   id<GADMediationNativeAdEventDelegate> delegate =
-                       strongSelf->_loadCompletionHandler(strongSelf, nil);
-                   strongSelf->_delegate = delegate;
-                 }
+      
+                 [strongSelf loadRequireData];
                }];
 }
 
-#pragma mark - GADMediationNativeAd
-
-- (GADNativeAdImage *)icon {
```

Comment:
As the previous comment was resolved, I am going to add another one.

Please keep this method as it's for mapping the icon asset, also see [the API documentation](https://developers.google.com/ad-manager/mobile-ads-sdk/ios/api/reference/Protocols/GADMediatedUnifiedNativeAd.html#icon).

We should do:
```
- (nullable GADNativeAdImage *)icon {
  return _icon;
}
```

Reply:
ok~